src/server.c

```diff
@@ -5757,7 +5757,8 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
             "mem_overhead_db_hashtable_rehashing:%zu\r\n", mh->overhead_db_hashtable_rehashing,
             "active_defrag_running:%d\r\n", server.active_defrag_running,
             "lazyfree_pending_objects:%zu\r\n", lazyfreeGetPendingObjectsCount(),
-            "lazyfreed_objects:%zu\r\n", lazyfreeGetFreedObjectsCount()));
+            "lazyfreed_objects:%zu\r\n", lazyfreeGetFreedObjectsCount(),
+            "defrag_supported: %s\r\n", defrag_supported));
```

Comment:
Maybe add a tcl test to verify the behavior.

Reply:
Updated in `other.tcl` file.